### DISCUSSION TITLE
urql not working with Graphql Instrumentation opentelemetry.

### DISCUSSION THREAD

**Discussion by Amangiri99** [urql-graphql/urql#3593] [Q&A]
Hi All,
In one of our services we are using urql as our graphql client. Everything was working as expected until we shifted to opentelemetry for observability.
As per the doc provided by one of it's instrumentation i.e. graphql instrumentation

> Note: graphql plugin instruments graphql directly. it should work with any package that wraps the graphql package (e.g apollo).

So I had a question does urql internally wraps the graphql package or it uses fetch?

**Comment by JoviDeCroock** [comment]
We use a different package named `@0no-co/graphql.web` which results in you having a lower bundle-size, ... You could do something like alias `@0no-co/graphql.web` to `graphql`. Not entirely sure what an opentelemetry setup on the front-end gives you of benefit though 😅

**Reply by Amangiri99** [reply]
Thanks @JoviDeCroock for suggesting the work around.
This is a backend service and for that we wanted to setup otel. Since this is a backend service shouldn't we use urql?

**Reply by JoviDeCroock** [reply]
Makes sense for a back-end service, aliasing is your best bet
